pennylane/devices/__init__.py

```diff
@@ -54,12 +54,17 @@
     Device
     DefaultMixed
     DefaultQubit
```

Comment:
Potentially could we just have
```suggestion
    DefaultQubit
    default_tensor.DefaultTensor
```
Might be worth checking if that works.

Reply:
It seems to work, although now there is an additional `default_tensor.` in front of the name. Still more elegant than before though!

![DT_attempt](https://github.com/user-attachments/assets/24ed96b3-d49b-454b-9d93-f7f4fcf52d3a)

Reply:
If there's no way to make this appear just as `DefaultTensor`, then this should be fine :). We have the devices page now! https://pennylane.ai/devices